src/redux.js

```diff
@@ -973,6 +981,35 @@ export function getIncorrectResults(state) {
   return getResultsByGrade(state, ResultsGrades.INCORRECT);
 }
 
+export function getAllResults(state) {
+  return getResultsByGrade(state, ResultsGrades.ALL);
+}
```

Comment:
Is this function different from `getAccuracyGrades` in the `accuracy.js` file?

Reply:
I believe so, because `getResultsByGrade` does conversion steps before filtering (via calls to `getConvertedAccuracyCheckExamples` and `getConvertedLabels`).